src/style.css

```diff
@@ -0,0 +1,156 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+}
+
+.card-apple {
+  height: 406px;
+  width: 198px;
```

Comment:
You can't set sizes for BEM blocks
And check checklist, you shouldn't hardcode the height of the card